cdap-docs/integrations/source/partners/cloudera/configuring.rst

```diff
@@ -41,8 +41,8 @@ the 'cdap' user installed by the parcel.
 Prerequisites
 =======================================
 
-#. Node.js (version 0.8.16 or greater) must be installed on the node(s) where the Web-App
-   role instance will run. You can download the latest version of Node.js from `nodejs.org
+#. Node.js (version from 0.8.16 through 0.10.36) must be installed on the node(s) where the Web-App
+   role instance will run. You can download the appropriate version of Node.js from `nodejs.org
    <http://nodejs.org>`__.
```

Comment:
You should give them the dist link; it's really hard to find on their website:

http://nodejs.org/dist/